tycho-bundles/org.eclipse.tycho.p2.resolver.impl/build.properties

```diff
@@ -15,3 +15,4 @@ bin.includes = META-INF/,\
                about.html,\
                plugin.properties
 source.. = src/main/java/
+additional.bundles = org.eclipse.pde.ds.lib
```

Comment:
this should really not be nessesary

Reply:
If the DS builder is enabled, it should add this automatically here and no warning should be shown.

Reply:
OK. I'll still push this one so we unbreak things for now. Once m2e/pde/whatever gets fixed let's remove it.

Reply:
You can better then add an import package (with optional) instead.

Reply:
Let's see what happens.